examples/platforms/nrf52811/radio.c

```diff
@@ -335,7 +335,7 @@ otError otPlatRadioReceive(otInstance *aInstance, uint8_t aChannel)
 
 otError otPlatRadioTransmit(otInstance *aInstance, otRadioFrame *aFrame)
 {
-    otError result = OT_ERROR_NONE;
+    bool result = true;
```

Comment:
It seems we should verify it's in receive state.

Reply:
Our radio driver can transmit from any state so check is not needed.